pkgs/applications/networking/feedreaders/feeds/default.nix

```diff
@@ -0,0 +1,108 @@
+{ lib
+, stdenv
+, fetchFromGitLab
+
+, appstream
+, gobject-introspection
+, meson
+, ninja
+, pkg-config
+, wrapGAppsHook
+
+, glib
+, gtk3
+, libhandy
+, webkitgtk
+, python3
+}:
+let
+  listparser = python3.pkgs.buildPythonPackage rec {
+    pname = "listparser";
+    version = "0.18";
+
+    src = python3.pkgs.fetchPypi {
+      inherit pname version;
+      sha256 = "0hdqs1mmayw1r8yla43hgb4d9y3zqs5483vgf8j9ygczkd2wrq2b";
+    };
+
+    propagatedBuildInputs = with python3.pkgs; [
+      requests
+      six
+    ];
+
+    doCheck = false;
```

Comment:
Could you add a comment why this is disabled?

Reply:
Default `checkPhase` doesn't seem to do anything (`Ran 0 tests..`). I'm not familiar with Python ecosystem at all, perhaps I'm just missing a `checkInputs` dependency but I simply don't know how to properly invoke the check phase so that tests are ran (the tests folder and its contents are present in the PyPI archive).

Reply:
Hmm, you might need to explicitly specify `checkPhase`.

The `README.md` for the tests specify that you can run them with either:

```
python -m unittest tests
```

or alternatively

```
mkdir build
cd build
meson ..
ninja test
```

Reply:
I think this is in context of `listparser`. I just added a commit that runs the tests based on what I found in `tox.ini`. The test suite however fails :-(. Maybe I'm missing something obvious but based on the output I really don't understand why.. Any ideas what's happening?

Reply:
NVM, I was looking at the development branch of the package instead of the tagged release, test should now run and pass with the commit I just pushed.